tfjs-backend-webgl/src/backend_webgl_test.ts

```diff
@@ -661,6 +670,9 @@ describeWithFlags('time webgl', WEBGL_ENVS, () => {
     // The tensor was already on gpu.
     expect(time.uploadWaitMs === 0);
     expect(time.downloadWaitMs === 0);
+    if (typeof time.kernelMs !== 'number') {
```

Comment:
This is in a tests, maybe just do `expect(time.kernelMs).toBeGreaterThan(0)` and let jasmine handle errors?